Source/SwiftLintBuiltInRules/Rules/Idiomatic/RedundantTypeAnnotationRule.swift

```diff
@@ -1,9 +1,120 @@
 import Foundation
-import SourceKittenFramework
+import SwiftLintCore
+import SwiftSyntax
 
-struct RedundantTypeAnnotationRule: OptInRule, SubstitutionCorrectableRule {
-    var configuration = SeverityConfiguration<Self>(.warning)
+struct RedundantTypeAnnotationRule: SwiftSyntaxCorrectableRule, OptInRule {
+    var configuration = RedundantTypeAnnotationConfiguration()
```

Comment:
This var can't be private because it's part of the [`Rule`](https://github.com/realm/SwiftLint/blob/6366fb85669c7a3e487fffea80285bafdccb7a8c/Source/SwiftLintCore/Protocols/Rule.swift#L20) protocol